lib/fathead/python_six/parse.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-6 -*-
```

Comment:
Why 6 exactly? If you change this to utf-8, it will run without errors in Python 3 too.